JSQMessagesViewController/Model/JSQVideoMediaItem.h

```diff
@@ -51,6 +56,6 @@
  *  isReadyToPlay. Once the video has been saved to disk, or is ready to stream, you can
  *  set the fileURL property or isReadyToPlay property, respectively.
  */
-- (instancetype)initWithFileURL:(NSURL *)fileURL isReadyToPlay:(BOOL)isReadyToPlay;
+- (instancetype)initWithFileURL:(NSURL *)fileURL withThumbnail:(UIImage*)thumbnail isReadyToPlay:(BOOL)isReadyToPlay;
```

Comment:
- we need to leave the original initializer.
- your new init should be:

```objective-c
- (instancetype)initWithFileURL:(NSURL *)fileURL isReadyToPlay:(BOOL)isReadyToPlay thumbnail:(UIImage *)thumbnail;
```

- then the original init should call this one with a `nil` thumbnail.
- need docs for the new initializer